docs/resources/cdm_link.md

```diff
@@ -0,0 +1,145 @@
+  - **Link to OSS on Alibaba Cloud**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0081.html)
+  - **Link to KODO/COS**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0087.html)
+  - **Link to HDFS**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0032.html)
+  - **Link to HBase**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0033.html)
+  - **Link to CloudTable**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0085.html)
+  - **Link to Hive**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0034.html)
+  - **Link to an FTP or SFTP Server**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0036.html)
+  - **Link to MongoDB**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0072.html)
+  - **Link to Kafka**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0035.html)
+  - **Link to DIS**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0075.html)
+  - **Link to Elasticsearch/Cloud Search Service**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0076.html)
+  - **Link to DLI**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0070.html)
+  - **Link to CloudTable OpenTSDB**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0089.html)
+  - **Link to Amazon S3**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0094.html)
+  - **Link to DMS Kafka**: [configuration detail](https://support.huaweicloud.com/intl/en-us/api-cdm/cdm_02_0095.html)
+
+-> Please remove the `linkconfig.` in the parameter key listed in the document. Configuration parameters such as
+ `password`, `ak`, `sk`, `accessKey` and `securityKey` do not need to be specified in `config`, they are specified in
+  the following parameters.
+
+* `password` - (Optional, String) Specifies the password for accessing the data sources.
+
+* `access_key` - (Optional, String) Specifies access key for accessing the data sources.
+  
+* `security_key` - (Optional, String) Specifies security key for accessing the data sources.
+
+* `enabled` - (Optional, Bool) Specifies whether to enable the link. The default value is `true`.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID in format of **cluster_id/link_name**. It is composed of the ID of CDM cluster and the name
+ of job, separated by a slash.
+
+## Import
+
+The link can be imported by `id`,It is composed of the ID of CDM cluster and the name of job, separated by a slash.
+ For example,
+
+```
+terraform import huaweicloud_cdm_link.test b11b407c-e604-4e8d-8bc4-92398320b847/linkName
```

Comment:
some fields cannot be imported, and descriptions should be added, please refer to: [/docs/resources/dcs_instance.md](https://github.com/huaweicloud/terraform-provider-huaweicloud/blob/master/docs/resources/dcs_instance.md)